website/project/model.py

```diff
@@ -75,16 +74,9 @@ def has_anonymous_link(node, auth):
     :param str link: any view-only link in the current url
     :return bool anonymous: Whether the node is anonymous to the user or not
     """
-    view_only_link = auth.private_key or request.args.get('view_only', '').strip('/')
```

Comment:
Added in 602f4738eac8c6f6ecd4927159cdccd04164542d:

website/addons/dataverse/views/crud.py
website/addons/figshare/views/crud.py
website/addons/github/views/crud.py
website/addons/osffiles/views.py
website/addons/wiki/views.py